pom.xml

```diff
@@ -41,7 +41,7 @@
 				<activeByDefault>true</activeByDefault>
 			</activation>
 			<properties>
-				<rosetta.dsl.version>7.7.3</rosetta.dsl.version>
+				<rosetta.dsl.version>7.8.0</rosetta.dsl.version>
 				<rosetta.bundle.version>6.5.3</rosetta.bundle.version>
```

Comment:
You probably also want to bump this version to whatever version you'll release the bundle with (6.6.0?)

Reply:
Have updated